macros/insert_into_metadata_table.sql

```diff
@@ -34,3 +34,5 @@
 
 {%- endmacro %}
 
+{% macro default__insert_into_metadata_table(database_name, schema_name, table_name, content) -%}
+{%- endmacro %}
```

Comment:
Is this needed @ahmedrad ?

Reply:
we have a project that uses both redshift and snowflake connectors. the redata package and models are only enabled on snowflake via the config but the compilation throws an error all together because this default method was missing. i dont think its a graceful way to fail on an unsupported connector and I cant find another way to install the package on a project that has some redshift models without failing compilation. open to other suggestions though!

Reply:
Makes total sense @ahmedrad ! Thanks for explaining.